pennylane/circuit_graph.py

```diff
@@ -117,15 +117,16 @@ def __init__(self, ops, obs, wires, par_info=None, trainable_params=None):
             op.queue_idx = k  # store the queue index in the Operator
 
             if hasattr(op, "return_type"):
-                if op.return_type is State:
+                meas_wires = wires or None  # cannot use empty wire list in MeasurementProcess
+                if op.return_type is qml.measurements.State:
                     # State measurements contain no wires by default, but wires are
                     # required for the circuit drawer, so we recreate the state
                     # measurement with all wires
```

Comment:
So the circuit drawer no longer relies on `CircuitGraph`.  I wonder if this is now dead code.

Reply:
I removed the code in the last commit. Let me know what you think!